ctapipe/instrument/camera/geometry.py

```diff
@@ -617,14 +617,15 @@ def from_table(cls, url_or_table, **kwargs):
 
     def __repr__(self):
         return (
-            "CameraGeometry(camera_name='{camera_name}', pix_type={pix_type!r}, "
-            "npix={npix}, cam_rot={camrot}, pix_rot={pixrot})"
+            "CameraGeometry(camera_name='{camera_name}', pix_type={pix_type}, "
+            "npix={npix}, cam_rot={camrot:.3f}, pix_rot={pixrot:.3f}, frame={frame})"
         ).format(
             camera_name=self.camera_name,
             pix_type=self.pix_type,
             npix=len(self.pix_id),
             pixrot=self.pix_rotation,
             camrot=self.cam_rotation,
+            frame=self.frame if self.frame is not None else "None",
```

Comment:
Shouldn't `self.frame` just work? `{frame}` should produce `None` `if frame is None`, right?

Reply:
yes, that was a holdover from the previoes commit that I forgot to clean up (before I only showed `self.frame.__class__.__name__`, but it's better to see the whole repr. I'll remove this check